src/components/user-dropdown-disclosure/index.tsx

```diff
@@ -67,6 +68,13 @@ const UserDropdownDisclosure = ({
 	listPosition,
 	user,
 }: UserDropdownDisclosureProps) => {
+	const { pathname } = useRouter()
+	// TODO handle elsewhere?
+	const shouldRenderThemeSwitcher =
+		__config.flags.enable_dark_mode &&
+		pathname !== '/' &&
+		pathname !== 'sign-up'
```

Comment:
💭 I wonder if we codify this in config or something, the routes that don't support theming.

Reply:
I think that's an amazing idea! 🙌🏻  I believe it was @BRKalow who did some similar refactoring many months ago, and created `__config.dev_dot.non_product_mobile_menu_routes`. We could name it something similar like `non_themed_routes`.

Reply:
I think i'll do this in a follow up PR!

Reply:
https://app.asana.com/0/1203652232454021/1204323512217789